test-resources/headless-core-stub/lambdas/mock-jwks/src/services/cache-jwk.ts

```diff
@@ -0,0 +1,33 @@
+import { randomUUID } from "crypto";
+import { JSONWebKeySet, JWK } from "jose";
+import { getJwkKeyPair, GetJwkKeyPairOptions } from "../../../../utils/src/keypair";
+import { getHashedKid } from "../../../../utils/src/hashing";
+import { ClientConfiguration } from "../../../../utils/src/services/client-configuration";
+import { CORE_STUB_SIGNING_PUBLIC_JWK } from "../constants";
+
+let cachedKeys: { jwks: JSONWebKeySet } | null = null;
+
+export const generateJWKS = async (clientId: string): Promise<{ jwks: JSONWebKeySet; privateKeys?: JWK[] }> => {
+    if (cachedKeys) {
+        return cachedKeys;
+    }
+
+    const ssmParameters = await ClientConfiguration.getConfig(clientId);
```

Comment:
Ah so I guess this would be a good use case for a getParameter function that just gets one param rather than all of them?